Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -7096,6 +7098,9 @@ def __init__(self, base_url, verify=True, proxy=False, ok_codes=tuple(), headers
             if not verify:
                 skip_cert_verification()
 
+            # removing trailing = char from env var value added by the server
+            self.timeout = float(os.environ.get('REQUESTS_TIMEOUT.' + get_integration_name(), '')[:-1] or os.environ.get('REQUESTS_TIMEOUT', '')[:-1] or self.REQUESTS_TIMEOUT)  # noqa: E501
```

Comment:
Add the timeout to the constructor too, so integrations that implement a param for controlling timeout can pass this in.

Reply:
Small cleanup use `os.getenv` so it is consistent with our other code sections that lookup env vars.

Reply:
Also, not clear why you have `[:-1]`? It is removing the last char

Reply:
We should also support scripts that may use the client. So maybe modify `get_integration_name` or create a new function that works with scripts. For scripts we need: `context.ScriptName`